### ISSUE TITLE
Support for Folder Color

### ISSUE THREAD

**Issue by costales** [snwh/suru-icon-theme#100]
Hi,

As these icons will be the default icons in Ubuntu, could be possible to extend the colors for supporting [Folder Color](http://foldercolor.tuxfamily.org/)? ([How to do it](https://bazaar.launchpad.net/~costales/folder-color/trunk/view/head:/README)).

Thanks in advance!

**Comment by snwh** [comment]
I'll think about it :wink: (it's a couple hundred extra icons in Suru's case)

**Comment by costales** [comment]
Thanks Sam. I have a script which will automatize the icons, even the color
inside, but you need the .svg for editing the color codes inside the files.
If you want to try it, I can upload it

**Comment by costales** [comment]
Hi Sam,

[This script](https://pastebin.com/1xxgsaGD) would generate your icons by itself, you only should need to choose the colors (you would need the .svg, not the .png), changing a hex color in the original file by another.

If you have any problem or doubt with the script, please, ping me here or by telegram @costales 

Best regards.

**Comment by snwh** [comment]
Right, however the Suru icon set uses PNG assets as the final icons. Just like when I added Folder Colors support to my Paper icon set ;)

It's a matter of me duplicating the source of the folders and recolouring them for each variant

**Comment by skewty** [comment]
Is anyone here willing to donate a bit of money (perhaps at [BountySource](https://www.bountysource.com/)) to help persuade (even if just a little) @snwh into taking on the boring, unrewarding task of making the icons?
